spec/requests/citizens/cash_outgoings_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe Citizens::CashOutgoingsController, type: :request do
+  let(:legal_aid_application) { create :legal_aid_application, :with_applicant, :with_non_passported_state_machine, :applicant_entering_means }
+  let!(:child_care) { create :transaction_type, :child_care }
+  let!(:maintenance_out) { create :transaction_type, :maintenance_out }
+  let(:next_flow_step) { flow_forward_path }
+
+  describe 'GET /citizens/cash_outgoings' do
```

Comment:
nitpicky me again. /citizens/cash_outgoings should be /citizens/cash_outgoing without the 's' at the end